app/assets/javascripts/discourse/tests/acceptance/admin-user-index-test.js

```diff
@@ -63,6 +63,38 @@ acceptance("Admin - User Index", function (needs) {
     );
   });
 
+  test("shows the number of post edits", async function (assert) {
+    await visit("/admin/users/1/eviltrout");
+
+    assert.equal(queryAll(".post-edits-count .value").text().trim(), "6");
+
+    assert.ok(
+      exists(".post-edits-count .controls .btn.btn-icon"),
+      "View Edits button exits"
+    );
+  });
+
+  test("a link to view post edits report exits", async function (assert) {
+    await visit("/admin/users/1/eviltrout");
+
+    let filter = encodeURIComponent('{"editor":"eviltrout"}');
+
+    assert.equal(
+      queryAll(".post-edits-count .controls .btn.btn-icon").attr("href"),
+      `/admin/reports/post_edits?filters=${filter}`,
```

Comment:
Instead of testing for the URL of the link, I think a better test will just be to click the button and ensure we land on the right route.